doc/reference/trd105-time.md

```diff
@@ -0,0 +1,455 @@
+Kernel Time HIL
+========================================
+
+**TRD:** 105<br/>
```

Comment:
IIRC usually draft TRDs won't get a TRD number. It might be just fine in this case if we are committed to finalize it once the single set of changes have landed, but I suspect if we follow due process already we might as well get it entirely correct.

Reply:
I thought we agreed we were fine with a number in this case, since we're already in agreement about it.

Reply:
I understood that in this case we were okay with giving it a number since we all had verbal agreement on this, too

Reply:
Yeah, so far the process has been that we add a TRD number when it starts getting close to finalization/when it's in a complete state. I think it's OK to assign 105 here. The thing we want to avoid is holes in the TRD numbering.